cognite/extractorutils/configtools/elements.py

```diff
@@ -696,3 +696,37 @@ def create_state_store(
             return LocalStateStore(file_path="states.json", cancellation_token=cancellation_token)
         else:
             return NoStateStore()
+
+
+class RegExpFlag(Enum):
+    IGNORECASE = "i"
+    ASCII_ONLY = "a"
```

Comment:
I really don't like making users configure fields with somewhat mystical `i` or `a` flags.

We should be using more human-readable fields